test/Microsoft.ComponentDetection.Orchestrator.Tests/MinimatchTests.cs

```diff
@@ -0,0 +1,197 @@
+namespace Microsoft.ComponentDetection.Orchestrator.Tests;
+
+using System;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+[TestClass]
```

Comment:
Thank you for including test coverage 😄 If you're up for it, I think we could get a pretty solid test coverage percentage by using some of the tests below:
- https://github.com/isaacs/minimatch/blob/main/test/basic.js
- https://github.com/isaacs/minimatch/blob/main/test/class-edge-cases.js
- https://github.com/isaacs/minimatch/blob/main/test/tricky-negations.js

Reply:
Thanks for the additional tests!